docker-compose.yml

```
@@ -0,0 +1,15 @@
+version: '3.7'
+
+services:
+
+  frontend:
+    build: oje
+    volumes:
+      - './:/app'
```

```suggestion
      - './app'?
```
or please explain why this line should not be changed?